values.yaml

```diff
@@ -18,6 +18,24 @@ agent:
     requests:
       cpu: 600m
       memory: 968Mi
+  nodeSelector:
+    beta.kubernetes.io/instance-type: t3.medium
+
+highmemagent:
+  replicaCount: 1
+  image:
+    repository: jetbrains/teamcity-agent
+    tag: latest
+    pullPolicy: Always
+  resources:
+    limits:
+      cpu: 800m
+      memory: 968Mi
+    requests:
+      cpu: 600m
+      memory: 968Mi
```

Comment:
I would directly set high requests/limits. A sensible default would be 16Gi I think.

Reply:
Yes, okay. It was kind of to force the user to overwrite as its a lot of memory, but i agree i makes more sense. I changed it